net/squid4/Portfile

```diff
@@ -5,6 +5,7 @@ PortGroup cxx11 1.1
 
 name            squid4
 version         4.4
+revision        1
```

Comment:
No revision increase should be done when fixing a build failure.

Reply:
Interesting. How to diffenrentiate between the Portfile versions?

Loooking into the MacPorts Guide [5.4. Global Keywords](https://guide.macports.org/#reference.keywords) there are:

1. version - The version of the ported software. _(not the version of the port)_
1. revision - used to track port revisions.
1. epoch - must be used when a port is updated to a version that is numerically less than the previous version

What did I miss?